### DISCUSSION TITLE
Feature idea: write Powerpoint programs

### DISCUSSION THREAD

**Discussion by rikhuijzer** [ASML-Labs/PPTX.jl#29] [Ideas]
Powerpoint programming sounds pretty ridiculous and I guess it is, but maybe there is something useful to learn from this CS course: https://youtu.be/_3loq22TxSc.

Maybe PPTX.jl should be able to generate stand alone applications?

**Comment by matthijscox-asml** [comment]
Oh my goodness. Sure, if someone wants to spend their time on this, but I would rather that they polish a few other aspects first :)
